sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java

```diff
@@ -6,20 +6,22 @@
 import com.azure.core.annotation.ReturnType;
 import com.azure.core.annotation.ServiceClient;
 import com.azure.core.annotation.ServiceMethod;
+import com.azure.core.credential.AccessToken;
```

Comment:
Please ignore this file - got included due to bad git branch structuring - will be fixed as soon as the 25146 PR is merged, this will become clean